packages/acceptance/packaging

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -p ${BOSH_INSTALL_TARGET}/src
+cp -a . ${BOSH_INSTALL_TARGET}/src
+
+pushd "${BOSH_INSTALL_TARGET}"
+  source /var/vcap/packages/golang-*-linux/bosh/compile.env
+
+  pushd src/acceptance/vendor
+    go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
hmmm this will install from the internet I think. it might be in the vendor folder so just a build like whats in the test script.